core/Transaction.go

```diff
@@ -0,0 +1,98 @@
+package core
+
+import (
+	"github.com/NethermindEth/juno/core/felt"
+)
+
+// Todo: define contract class
+type ContractClass struct{}
+
+type DeployTransaction struct {
```

Comment:
`deploy` is depreciated. We should use `deploy syscall` that's explained [here](https://docs.starknet.io/documentation/develop/Contracts/contract-classes/) instead.